rs/nns/governance/src/neuron/types.rs

```diff
@@ -480,6 +469,10 @@ impl Neuron {
         }
     }
 
+    pub(crate) fn refresh_voting_power(&mut self, now_seconds: u64) {
+        self.voting_power_refreshed_timestamp_seconds = now_seconds;
+    }
```

Comment:
I'm not exactly sure what it means to refresh voting power or why it needs to happen in some circumstances, but is the whole "refresh_voting_power" operation just updating a timestamp?